scripts/delayed.js

```diff
@@ -1,10 +1,22 @@
 // eslint-disable-next-line import/no-cycle
-import { loadCSS, sampleRUM } from './lib-franklin.js';
+import { loadCSS, loadScript, sampleRUM } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+async function loadAdobeLaunch() {
+  const adobedtmSrc = 'https://assets.adobedtm.com/9273d4aedcd2/6eb97addd328/launch-1dd947b3f935.min.js';
+
+  await loadScript(adobedtmSrc, {
+    type: 'text/javascript',
+    async: true,
+  });
+}
+
 store.emit('delayed:loaded');
 
 loadCSS(`${window.hlx.codeBasePath}/styles/icons.css`);
+
+await loadAdobeLaunch();
```

Comment:
I would avoid top-level `await` because support is still [relatively low](https://caniuse.com/?search=top%20level%20await), otherwise looks good

Reply:
Thanks @maxakuru. I removed the `await`.